independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/workspace/LocalProject.java

```diff
@@ -28,17 +28,24 @@
 
     private static final String PROJECT_BASEDIR = "${project.basedir}";
     private static final String POM_XML = "pom.xml";
+    private static final String REVISION = "revision";
```

Comment:
Just a heads up: https://maven.apache.org/maven-ci-friendly.html does not only support `revision` but also `sha1` and/or `changelist`.

Reply:
Thanks for pointing this out. I'll probably add support for those in another PR just to get this one in quicker. Unless you tell me you absolutely need them :)

Reply:
I'm fine with just `revision`!